examples/burial-form/utils.ts

```diff
@@ -0,0 +1,9 @@
+import { isBefore } from 'date-fns';
```

Comment:
decided to add date-fns to our package.json and use their `isBefore` validator for evaluating two dates instead of having us roll our own.